modin/pandas/test/utils.py

```diff
@@ -39,9 +39,21 @@
     "Big": (2 ** 7, 2 ** 12),
 }
 
+# dataset sizes for IO operations
+DATASET_SIZE_DICT_IO = {
+    "Small": 64,
+    "Normal": 2000,
+    "Big": 20000,
+}
+
```

Comment:
You also have a similar variable in `modin/pandas/test/test_io.py` - `DATASET_SIZE_DICT`. Can we reuse it?

Reply:
Yes, i think we can use number of rows from `DATASET_SIZE_DICT`, default number of rows is larger, but tests execution time ingresses negligible.